CHANGELOG.rst

```diff
@@ -1,5 +1,9 @@
 .. _changelog:
 
+0.4x
```

Comment:
Call it 0.42.0. Every new commit on renku has to be releasable without further changes. 

The commit you merged is a feature so it will make a new release.